### DISCUSSION TITLE
[firebase_storage] getActiveUploadTasks not available in Flutter

### DISCUSSION THREAD

**Discussion by ghost** [firebase/flutterfire#5570] [Feature request]
Not sure if this is a bug or simply not been implemented yet. But this seems like a major flaw in the api right now. 

I'm trying to track multiple uploads, and unavailable to do so until getActiveUploadTasks is available on the Flutter version. 

Is there a work-around or a different name to it? Such a critical call to detect the performance and status of all the uploaded files.

**Comment by iapicca** [comment]
Hi @EricPreventa 
could you please provide your updated  `flutter doctor -v`?
Thank you

**Comment by ghost** [comment]
[✓] Flutter (Channel stable, v1.9.1+hotfix.6, on Mac OS X 10.14.6 18G1012,
    locale en-US)
    • Flutter version 1.9.1+hotfix.6 at /Users/hasseye/Development/flutter
    • Framework revision 68587a0916 (8 weeks ago), 2019-09-13 19:46:58 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0

[!] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/hasseye/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Users/hasseye/Library/Application
      Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/191.5977832/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor
      --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.0, Build version 11A420a
    • CocoaPods version 1.8.4

[!] Android Studio (version 3.5)
    • Android Studio at /Users/hasseye/Library/Application
      Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/191.5977832/Android
      Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[✓] Connected device (1 available)
    • iPhone 11 Pro Max • 450802A7-52D8-46BF-A23A-4E2956A892F3 • ios •
      com.apple.CoreSimulator.SimRuntime.iOS-13-0 (simulator)

**Comment by ghost** [comment]
Although not sure what my flutter doctor will say. I'm more so requesting a feature that's available in some of the APIs, but not available in the Flutter package yet.

**Comment by iapicca** [comment]
Hi @EricPreventa 
I believe you are correct pointing this issue more as enhancement rather than a bug.
Thank you

**Comment by ghost** [comment]
This is available in the API, just hasn't been opened up for Flutter users yet. I'm hoping that since it's minimal work needed, the next Flutter API release could expose this.

Thank you
